lib/shoryuken/client.rb

```diff
@@ -9,15 +9,15 @@ def queues(name)
       end
 
       def sns
-        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns_endpoint))
+        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns))
       end
 
       def sns_arn
         @sns_arn ||= SnsArn
       end
 
       def sqs
-        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs_endpoint))
+        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs))
```

Comment:
@atyndall this is a break change for people using `sqs_endpoint`, that would require a major release version.